.env.example

```diff
@@ -3,7 +3,7 @@
 #####################
 
 REACT_PORT=3000  # local port for frontend
-NODE_PORT=3001  # local port for backend
+BACKEND_PORT=3001  # local port for backend
```

Comment:
Not `FLASK_PORT`?
I think its better because the backend is a bit more than just the flask app, it's all the things which are running in the *back* (like flask and mongo)

Reply:
You're right, but I'll change it to `FLASK_RUN_PORT` then because `flask` read it by default. It'll solve the PORT configuration choice I discussed in the Issue desc.